### DISCUSSION TITLE
smplayer installation location

### DISCUSSION THREAD

**Discussion by joeboy8** [smplayer-dev/smplayer#358] [Q&A]
Where does smplayer install in Windows 10? I can't find it, and I want to create a desktop shortcut.

**Comment by smplayer-dev** [comment]
Usually in  C:\Program Files\SMPlayer or C:\Program Files (x86)\SMPlayer

**Reply by joeboy8** [reply]
Yes, I had found it....problem is, it is in Program Files\WindowsApps\55144Ricard..., which I can't access

**Reply by smplayer-dev** [reply]
Applications installed from the Windows Store are hidden.
This explains how to create a shortcut:
https://www.thurrott.com/windows/windows-10/5328/windows-10-tip-create-shortcuts-for-universal-apps
